js/packages/teams-ai/src/authentication/MessagingExtensionAuthenticationBase.ts

```diff
@@ -0,0 +1,95 @@
+import { ActivityTypes, InvokeResponse, TokenResponse, TurnContext } from 'botbuilder';
+import { MessagingExtensionsInvokeNames } from '../MessageExtensions';
+
+/**
+ * @internal
+ */
+export abstract class MessagingExtensionAuthenticationBase {
+    public async authenticate(context: TurnContext): Promise<string | undefined> {
```

Comment:
Sorry for the hassle, but please rename this file and **new** methods/variables/etc with MessageExtension instead of MessagingExtension. We will try to stay consistent about this from now on.